README.md

```diff
@@ -134,17 +134,41 @@ Redis is a data structures server so unlike in plain-key value stores in which s
 
 ## Installing the Module ##
 
-First clone this respository if you haven't already anywhere on your local system. To do this, first go into the directory you wish to clone the repository in and type into the command line "git clone https://github.com/cmsc22000-project-2018/redis-tries.git"
+First clone this respository if you haven't already anywhere on your local system. To do this, first go into the directory you wish to clone the repository and then type into the command line: 
 
-Inside the *module* directory, type "make all" into the command line to build the trie.so library.
+    git clone https://github.com/cmsc22000-project-2018/redis-tries.git
+
+Inside the repository, initialize the submodule by typing into the command line:
+
+    git submodule update --init --recursive
```

Comment:
Let's put a `$ ` in front of every shell command.

Reply:
Fixed